source/common/stats/hystrix.cc

```diff
@@ -5,32 +5,31 @@
 #include <iostream>
 #include <sstream>
 
+#include "common/buffer/buffer_impl.h"
 #include "common/common/logger.h"
 
+#include "absl/strings/str_cat.h"
+
 namespace Envoy {
 namespace Stats {
 
 const uint64_t Hystrix::DEFAULT_NUM_OF_BUCKETS;
 const uint64_t Hystrix::ROLLING_WINDOW_IN_MS;
 const uint64_t Hystrix::PING_INTERVAL_IN_MS;
 
-// add new value to rolling window, in place of oldest one
-void Hystrix::pushNewValue(std::string key, uint64_t value) {
-  // create vector if do not exist
-  if (rolling_stats_map_.find(key) == rolling_stats_map_.end()) {
-    rolling_stats_map_[key].resize(num_of_buckets_, value);
-  } else {
-    rolling_stats_map_[key][current_index_] = value;
-  }
+// Add new value to rolling window, in place of oldest one.
+void Hystrix::pushNewValue(const std::string& key, uint64_t value) {
+  // Create vector if do not exist.
+  rolling_stats_map_[key].resize(num_of_buckets_);
+  rolling_stats_map_[key][current_index_] = value;
```

Comment:
why did you change this? did you remove the "value" parameter on purpose? it was added there to fix a bug